src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueTests.cs

```diff
@@ -8077,5 +8077,403 @@ .maxstack  2
 }
 ");
         }
+
+        [Fact]
+        public void ForeachStatement_Reorder()
```

Comment:
Also, add ArrayOfTuple test to VB


---
In reply to: [115573899](https://github.com/dotnet/roslyn/pull/19283#discussion_r115573899) [](ancestors = 115573899)